src/target/riscv/riscv-013.c

```diff
@@ -2570,6 +2570,18 @@ static int execute_fence(struct target *target)
 	return ERROR_OK;
 }
 
+static void log_memory_access128(target_addr_t address, uint64_t value_h,
+		uint64_t value_l, bool read)
```

Comment:
Please, while making changes here, change also the name of the `read` argument to something else. That's because `read` is a system function (`man 2 read`):

```suggestion
		uint64_t value_l, bool is_read)
```

Reply:
I submitted a separate patch to rename variables